### ISSUE TITLE
Get Designer Input on BackdropCMS 'Out of the Box' appearance

### ISSUE THREAD

**Issue by stpaultim** [backdrop/backdrop-issues#5581]
## Description of the need

While the work on this issue: https://github.com/backdrop/backdrop-issues/issues/4903 was initially about adding a new content type to Backdrop CMS, the PR also incorporated some sample content to help illustrate the use of this new "hidden path" content type and also to make a default Backdrop CMS site look more like an actual website. 

During the discussion about this issue, a number of questions were raised about default "design" of Backdrop and how to make it better. 

1) Should we use different images
2) What is the right color scheme for the Out of the Box experience
3) Should we replace the current Hero image with something more "catchy". 

Very late in that discussion, we asked @wesruv for his input as a designer, which turned out to be a bit problematic because of how late it was in the process. A decision was made to proceed with the technical aspects of this PR along with a best effort attempt to select images and color scheme based upon existing feedback. 

The goal was to follow up with a fresh issue to review the entire front page (or event the entire site) with the help of a designer, but this time engaging the designer at the outset of the discussion. 

## Proposed solution

1) That we identify a designer that is willing and start over with us and review the look and feel of Backdrop with the goal towards a new PR that focuses on the visual aspects of Backdrop CMS right Out of the Box. That this designer work with a small sub-committee that is engaged in the process from start to finish.
2) That through a (short) series of meetings with the designer, we collect input input from the community on how to make Backdrop look better. 
3) That we work towards a PR that focuses on these visual elements. 

## Alternatives that have been considered

In recognition of the fact that making design choices in an Open Source project is destined to generate a lot of "bike-shedding" (time consuming arguments about small subjective details), we'd like to define a process at the outset and give extra weight to a designated designer - with the ultimate goal of avoiding a lot of last minute second guessing and an inability to make good design decisions. 

We also need to recognize that the Out of the Box experience for Backdrop is greatly shaped by the theme, in this case Basis. Our ability to make Backwards compatible changes to Basis is limited and that must be kept in mind. This group will need to figure out what elements can be changed and how best to accomplish this task.

**Comment by BWPanda** [comment]
So either we make changes to Basis, or we ask a designer to design a new default theme for Backdrop. I prefer the latter.

**Comment by indigoxela** [comment]
> The goal was to follow up with a fresh issue to review the entire front page ... identify a designer that is willing and start over with us and review the look and feel of Backdrop

Did that happen already, or are we still seeking designers, who are willing to dig into this?

If the latter - does this have impact on #4903? Is there anything that has to happen before next minor release?

**Comment by stpaultim** [comment]
> Did that happen already, or are we still seeking designers, who are willing to dig into this?

No. We haven't decided we want to do this yet. So, no effort has been made to seek out designers. 

> If the latter - does this have impact on https://github.com/backdrop/backdrop-issues/issues/4903? Is there anything that has to happen before next minor release?

No, the intent of this issue was to separate this discussion from #4903, so that these design decisions do not hold up that issue. The intent of this issue was to say, that if you are not happy with the design decisions in #4903, this is how we can fix them without blocking #4903. 

If this issue proceeds it can proceed independently of #4903 on it's own schedule.
